inscriptions/ISic003443.xml

```diff
@@ -193,13 +198,10 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI337248">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MB">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Συ<supplied reason="lost">— —</supplied>-<lb n="2"/>ου.
-                    <lb n="3"/>
+                    <lb n="1"/><w part="I">Συ</w><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
Following Cordano, it would seem reasonable to make Συ the first part of a name, and ου the final part of the following patronym (which itself must start on the end of the first line).